frontend/apps/web/src/pages/tests.tsx

```diff
@@ -344,7 +344,7 @@ const AllTests: React.FC = () => {
               },
               status: {
                 header: t('filecoin.allTests.status'),
-                Cell: ({ data }) => getButton(data.status),
+                Cell: ({ data }) => getButton(data.status, data.ciLink),
```

Comment:
I'd guess that there is more usage of getButton for testStatus than this.

maybe in BehaviorModal.tsx (there should be list of test)

Reply:
Done